array/DNAm/preprocessing/calcQCMetrics.r

```diff
@@ -201,6 +201,30 @@ if(!"pFilter" %in% colnames(QCmetrics)){
 	QCmetrics<-cbind(QCmetrics,"pFilter"= pFOut)
 }
 
+
+# detection p value and beacd count filtering for probe filtering
+if(!exists("probeFilt")){

```

Comment:
How would this variable already be defined by this point? I can only see this variable name being used in this file, `QC.rmd` and `normalisation.r`. Considering this script is run first out of the three, surely the variable can't have been declared yet? What am I missing?

Reply:
This would already be defined if a QC object that contains it is being loaded instead of a new QCmetrics object being created (calcQCmetrics.r lines 66:79). Similar to the snp object (lines 348:353)

Reply:
Aighty. I've never actually had a QCmetrics object of my own, so I don't really know what it includes.